Backend/dev/migrations/dynamic-offer-driver-app/0566-add-mocid-in-driver-location.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE atlas_driver_offer_bpp.driver_location ADD COLUMN merchant_operating_city_id TYPE text;
```

Comment:
no need to create multiple migrations for bpp, you can move all queries to single one